src/main/target/STM32F3DISCOVERY/target.h

```diff
@@ -32,7 +32,9 @@
 #define CONFIG_FASTLOOP_PREFERRED_ACC ACC_NONE
 
 #undef TELEMETRY_JETIEXBUS // ROM SAVING
-
+#undef CMS
+#undef BLACKBOX
+#undef USE_CLI
```

Comment:
Probably don't need to `undef` all these now.

Reply:
Tried to undef... something is still overflowing by 54k?  what could it be?

Reply:
I tried to make it from master and it fails as well